AccessibilityInsightsForAndroidService/app/src/main/java/com/microsoft/accessibilityinsightsforandroidservice/FocusVisualizerController.java

```diff
@@ -62,7 +69,7 @@ public void onOrientationChanged(Integer orientation) {
     if (focusVisualizationStateManager.getState() == false) {
       return;
     }
-
+    lastOrientationChange = new Date();
```

Comment:
Do we want to use the DateProvider for this as well? Or just use new Date() for the instance below? Is there another reason for doing it 2 different ways?